examples/docs_snippets/docs_snippets/concepts/partitions_schedules_sensors/sensors/sensors.py

```diff
@@ -233,47 +233,43 @@ def uses_db_connection():
 
 # end_build_resources_example
 
-
-@repository
-def my_repository():
-    return [my_job, log_file_job, my_directory_sensor, sensor_A, sensor_B]
+code_location_a = Definitions(
```

Comment:
variable names shouldn't be named after code locations. Just name them `defs`

Reply:
Nope; all good. I'll make the change.